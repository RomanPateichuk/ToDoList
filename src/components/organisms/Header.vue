<template>
  <header :class="$style.header">
    <h1 :class="$style.name">to do list</h1>
  </header>
</template>

<style lang="scss" module>
@import "@/assets/scss/main.scss";

.header {
  font-family: "Open Sans", sans-serif;
  background-color: $header;
  text-align: center;
  height: 3.125rem;

  .name {
    font-size: 1.25rem;
    line-height: 2.5rem;
  }
}
</style>
