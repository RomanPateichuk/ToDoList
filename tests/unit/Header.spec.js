import { mount } from "@vue/test-utils";
import Header from "../../src/components/organisms/Header.vue";

describe("Тестирование компонента Header:", () => {
  const wrapper = mount(Header);
  test("соответствие снимку", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
