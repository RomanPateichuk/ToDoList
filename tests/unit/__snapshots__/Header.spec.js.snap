// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тестирование компонента Header: соответствие снимку 1`] = `
<header>
  <h1>to do list</h1>
</header>
`;
